spanner/api/Spanner.Samples/UpdateInstanceConfigAsync.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_update_instance_config]
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Google.Cloud.Spanner.Admin.Instance.V1;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
+
+public class UpdateInstanceConfigAsyncSample
+{
+    public async Task UpdateInstanceConfigAsync(string projectId, string instanceConfigId)
```

Comment:
You may return the operation here and check that is successful on the test, for instance.